src/udiskslinuxfilesystemhelpers.c

```diff
@@ -105,7 +105,7 @@ gboolean take_filesystem_ownership (const gchar *device,
       else
         {
           /* device is not mounted, we need to mount it */
-          mountpoint = g_mkdtemp (g_strdup (PACKAGE_LOCALSTATE_DIR "/run/udisks2/temp-mount-XXXXXX"));
+          mountpoint = g_mkdtemp (g_strdup ("/run/udisks2/temp-mount-XXXXXX"));
```

Comment:
Can we nuke that macro altogether?

Reply:
We are using it for udisks state files on systems where we mount devices to `/media` instead of `/run/media`. I'm not sure if it still is relevant but I don't want do break more things, so I'd like to keep it that way.

Reply:
Fair enough, just please make sure the directory is created or checked to be existing in those places.